src/components/DetailsList.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-len */
+/* eslint-disable react/jsx-key */
+/* eslint-disable array-callback-return */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import style from './components.module.css';
```

Comment:
- [ ] Like the previous reviewer mentioned kindly avoid disabling linters they are in place to help you write more clean code. There are a lot of linters that have been disabled in different files. Kindly fix them 🙏